Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -76,19 +81,47 @@
             "visibleRule": "DeployToSlotFlag = true"
         },
         {
+            "name": "VirtualApplication",
+            "type": "string",
+            "label": "Virtual Application",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "Specify the name of the Virtual Application that has been configured in the Azure portal. The option is not required for deployments to the website root."
+        },
+        {
             "name": "Package",
             "type": "filePath",
-            "label": "Package",
+            "label": "Package or Folder",
             "defaultValue": "",
             "required": true,
             "helpMarkDown": "Location of the Web App zip package on the automation agent or on a UNC path accessible to the automation agent like, \\\\\\\\BudgetIT\\Web\\Deploy\\Fabrikam.zip. Predefined system variables and wild cards like, $(System.DefaultWorkingDirectory)\\\\***.zip` can be also used here."
```

Comment:
Folder or file path to the Web App package. Variables (Build | Release), wild cards are supported. <br/>
For example, $(System.DefaultWorkingDirectory)\\\\***.zip